nationtweaker.js

```diff
@@ -61,6 +66,27 @@ var tweaks = [
 		"suggestLink": "http://nationbuilder.com/ineffyble/include_in_top_nav_should_be_unticked_by_default_when_creating_new_pages",
 		"function": "stopNewPagesAddedToNavByDefault",
 		"matches": [paths.pageNew]
+	},
+	{
+		"name": "Increase size of email preview windows",
+		"description": "Make email previews full width and 600px high",
+		"suggestLink": "#",
```

Comment:
Nope, I never made suggestions for these since there are more significant issues I'd rather they fix first. I went with `#` because the links were coming up undefined without